code/game/objects/items/weapons/AI_modules.dm

```diff
@@ -351,15 +377,19 @@ obj/item/aiModule/solgov_aggressive
 /obj/item/aiModule/syndicate // Slightly more dynamic freeform module -- TLE
 	name = "hacked AI module"
 	var/newFreeFormLaw = ""
-	desc = "A hacked AI law module: '<freeform>'."
+	desc = "A hacked AI law module."
 	origin_tech = list(TECH_DATA = 3, TECH_MATERIAL = 6, TECH_ESOTERIC = 7)
 
 /obj/item/aiModule/syndicate/attack_self(var/mob/user as mob)
-	..()
 	var/newlaw = ""
 	var/targName = sanitize(input("Please enter a new law for the AI.", "Freeform Law Entry", newlaw))
 	newFreeFormLaw = targName
-	desc = "A hacked AI law module:  '[newFreeFormLaw]'."
+	..()
+
+/obj/item/aiModule/syndicate/show_laws(mob/user)
+	if(newFreeFormLaw)
+		to_chat(user, "<b>Programmed Law:</b>")
```

Comment:
Use macro

Reply:
Please stop commenting "use macro" everywhere, aaaaa.

Reply:
Have to maintain code style; if you're typing out HTML by hand you're probably doing it wrong